shared.lisp

```diff
@@ -21,6 +21,9 @@
 (defvar *completion-queue* nil "The global completion queue used to
 manage grpc calls.")
 
+(defvar *call-deadline* nil
+  "The deadline for a grpc call. The default, nil, specifies an infinite deadline")
```

Comment:
I don't think this is something that should be a defvar, more something you specify, and the user can make a defvar and set in their code...